### PR TITLE
:sparkles: New page : Mentions Legales

### PR DESCRIPTION
**Pull request by IncSow**

### CODE REVIEW

**Comment by amorfus78** [comment]
the href of terms and services in the footer is "/cgu" not "/legal"

**Comment by IncSow** [comment]
We lready have /cgu (TOS), this isn't the same thing :)

**Review by amorfus78 — DISMISSED** [review]
LGTM

File: front-end/src/locales/en.json [hunk 1]
```diff
@@ -149,6 +149,10 @@
     "slogan": "From the Highlands of Scotland, our furniture is immortal",
     "pageNotFound": "Page not found"
   },
+  "LEGAL": {
```
Comment (DenisMarine) [inline front-end/src/locales/en.json:152]: not in uppercase !

File: front-end/src/pages/legal.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Page from "@/components/layout/Page"
+import { t } from "i18next"
+
+const LegalPages = () => {
+  return (
+    <Page title="CGU">
```
Comment (DenisMarine) [inline front-end/src/pages/legal.jsx:6]: title CGU ????

File: front-end/src/pages/legal.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <div className="h-[80vh] flex justify-center items-center">
+        <div className="p-6 shadow-lg rounded-lg w-[80vw] h-min">
+          <h1 className="text-2xl font-semibold mb-4 text-center">
+            Mentions légales
```
Comment (DenisMarine) [inline front-end/src/pages/legal.jsx:10]: i18n !!!!

**Review by DenisMarine — CHANGES_REQUESTED** [review]

File: front-end/src/pages/legal.jsx (hunk 2 above)
Comment (DenisMarine) [reply front-end/src/pages/legal.jsx]: and need to apply i18n on the title !

**Review by DenisMarine — APPROVED** [review]

**Pull request merged**